src/network/sbfTcpConnection.c

```diff
@@ -1,9 +1,9 @@
 #include "sbfTcpConnection.h"
 #include "sbfTcpConnectionPrivate.h"
 
-#include <sys/socket.h>
-#include <netinet/in.h>
-#include <netinet/tcp.h>
+//#include <sys/socket.h>
```

Comment:
Delete rather than comment out

C

Reply:
Completely missed reviewing these, as they were moved (couldn't see the diff). I will remove these, as they're included in the compat libraries for linux and darwin.